datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -492,6 +492,32 @@ CREATE TABLE dcsa_im_v3_0.event_classifier (
     event_classifier_description varchar(250) NULL
 );
 
+DROP TABLE IF EXISTS dcsa_im_v3_0.equipment_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.equipment_event_type (
+    equipment_event_type_code varchar(4) PRIMARY KEY,
+    equipment_event_type_name varchar(100) NOT NULL,
+    equipment_event_type_description varchar(250) NOT NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.shipment_information_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.shipment_information_type (
+    shipment_information_type_code varchar(3) PRIMARY KEY,
+    shipment_information_type_name varchar(100) NOT NULL,
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.transport_event_type CASCADE;
+CREATE TABLE dcsa_im_v3_0.transport_event_type (
+    transport_event_type_code varchar(4) PRIMARY KEY,
+    transport_event_type_name varchar(100) NOT NULL,
+    transport_event_type_description varchar(250) NOT NULL
+);
+
+DROP TABLE IF EXISTS dcsa_im_v3_0.empty_indicator CASCADE;
```

Comment:
I am quite sure there is a usage of this somewhere in our model - can you please add a REFERENCES..